styleguide/source/_patterns/01-molecules/09-search-results/search-results-best-match-event.twig

```diff
@@ -0,0 +1,29 @@
+{#
+ # @file search-results-best-match-event.twig
+ # Copyright 2017 Palantir.net, Inc.
+ #}
+
+<div class="layout layout_one_up">
+  {% include 'atoms-h6' with { 'class' : 'search-result_label', 'content' : 'Best Match' } %}
+  <div class="search-result">
```

Comment:
Additionally, there are some funky hover states going on. For example when you hover just the box, it's background changes, but then if you hover the text, it's color changes independently. It should be one hover state when you hover on the block